plugins/sqlfluff-plugin-example/src/example/rules.py

```diff
@@ -37,8 +37,10 @@ def get_configs_info() -> dict:
     }
 
 
-@document_fix_compatible
+# These two decorators allow plugins
+# to be displayed in the sqlfluff docs
 @document_configuration
+@document_fix_compatible
```

Comment:
Nitpick: Is there a reason for moving `@document_fix_compatible` so it follows `@document_configuration` rather than preceding it?

Reply:
I thought it did, but it didn't! I swapped it back